packages/cardano-services-client/src/version.ts

```diff
@@ -1,5 +1,5 @@
-// auto-generated using /home/rhys/code/iohk/cardano-js-sdk/packages/cardano-services-client/scripts/createVersionSource.js
+// auto-generated using /Users/vanessa/Documents/Projects/IOHK/DaedalusLace/cardano-js-sdk/packages/cardano-services-client/scripts/createVersionSource.js
```

Comment:
Whoops, I'll fix this absolute path in my local branch

Reply:
I think it would be better to just say `// auto-generated using cardano-services-client/scripts/createVersionSource.js`

Reply:
It's fixed in https://github.com/input-output-hk/cardano-js-sdk/pull/781